apim-lab/2-developer-portal/developer-portal-2-1-publishing.md

```diff
@@ -10,13 +10,13 @@ nav_order: 1
 
 The APIM Developer Portal uses role-based access control (RBAC). After creating the APIM instance, you need to first **access the Developer Portal using the admin role**, which will trigger its activation.
 
-- Click the *Developer portal* link in the *Overview* blade of the APIM instance:
+- Click the **Developer portal** link in the **Overview** blade of the APIM instance:
 
   ![APIM Developer Portal](../../assets/images/apim-developer-portal.png)
 
   ![APIM Developer Portal Admin Launch](../../assets/images/apim-developer-portal-admin-launch.png)
 
-- You can now proceed with publishing the Developer Portal to allow anonymous and authenticated roles. Back in the Azure Portal, locate _Developer Portal_ in the left-hand resource menu, then click on *Portal overview*. Press the *Publish* button and confirm with *Yes* in the popup. The portal will then be available for users to access.
+- You can now proceed with publishing the Developer Portal to allow anonymous and authenticated roles. Back in the Azure Portal, locate **_Developer Portal_** in the left-hand resource menu, then click on **Portal overview**. Press the **Publish** button and confirm with **Yes** in the popup. The portal will then be available for users to access.

```

Comment:
Check bold/italic for this, please: `**_Developer Portal_**`

Reply:
Yes, use bold only for clarity yes.